### PR TITLE
Remove Python 3.7

### LINKED ISSUE #274: Remove Python 3.7 (EOL)
Python 3.7 is now end-of-life, so will no longer be supported by cfdm.

### PR DESCRIPTION
**Pull request by davidhassell**
Fixes #274

### CODE REVIEW

**Comment by sadielbartholomew** [comment]
Hi David, is this PR meant to be in draft form? Can it be reviewed?

**Comment by davidhassell** [comment]
Oops

File: setup.py [hunk 1]
```diff
@@ -161,18 +161,18 @@ def _get_version():
         "climate",
     ],
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 5 - Production/Stable",
```
Comment (sadielbartholomew) [inline setup.py:164]: Haha, oops - good spot. This would be a very mature beta version :laughing:

**Review by sadielbartholomew — APPROVED** [review]
Perfect, please merge.

**Pull request merged**